content/docs/gke/deploy/deploy-cli.md

```diff
@@ -32,92 +32,130 @@ Before installing Kubeflow on the command line:
     # description in the text below this code snippet.
-    # For example,  'kubeflow-test' or 'kfw-test'.
+    # For example, your app name can be  'kubeflow-test' or 'kfw-test'.
     export KFAPP=<your choice of application directory name>
 
-    export ZONE=<your target GCP zone> # where the deployment will be created
-    export PROJECT=<your GCP project ID>
+    # The following command is optional. It adds the kfctl binary to your path.
+    # If you don't add kfctl to your path, you must use the full path
+    # each time you run kfctl.
+    export PATH=$PATH:<path to your kfctl file>
+    ```
 
-    # Run the following commands for the default installation which uses Cloud IAP:
-    export CONFIG="{{% config-uri-gcp-iap %}}"
-    kfctl init ${KFAPP} --project=${PROJECT} --config=${CONFIG} -V
-    # Alternatively, run these commands if you want to use basic authentication:
-    export CONFIG="{{% config-uri-gcp-basic-auth %}}"
-    kfctl init ${KFAPP} --project=${PROJECT} --config=${CONFIG} -V --use_basic_auth
+<a id="set-up-and-deploy"></a>
+## Set up and deploy Kubeflow
+
+To set up and deploy Kubeflow using the **default settings**,
+run the **apply** command:
+
+```
+mkdir ${KFAPP}
+cd ${KFAPP}
+kfctl apply -V -f ${CONFIG}
+```
+
+Alternatively, you can set up your configuration files without deploying 
+Kubeflow, so that you can customize the configuration before deployment:
+
+1. Run the **build** command to set up your configuration:
+
+  ```
+  mkdir ${KFAPP}
+  cd ${KFAPP}
+  kfctl build -V -f ${CONFIG}
+  ```
+
+1. Edit the configuration files, as described in the guide to
+  [customizing your Kubeflow deployment](/docs/gke/customizing-gke).
+1. When you have finished editing the configuration, run the **apply**
+  command to deploy Kubeflow:
+
+  ```
+  kfctl apply -V -f ${CONFIG}
+  ```
+
+Notes:
+
+* **${KFAPP}** - The _name_ of a directory where you want Kubeflow 
+  configurations to be stored. 
+  If you want a custom deployment name, specify that name here.
+  The value of this variable becomes the name of your deployment.
+  The value of KFAPP must consist of lower case alphanumeric characters or
+  '-', and must start and end with an alphanumeric character.
+  For example,  `kubeflow-test` or `kfw-test`.
+  The value of this variable cannot be greater than 25 characters. It must
+  contain just the directory name, not the full path to the directory.
+  The content of this directory is described in the next section.
+* **${PROJECT}** - The project ID of the GCP project where you want Kubeflow 
+  deployed.
+* **${ZONE}** - The GCP zone where you want to create the Kubeflow deployment.
+  You can see a list of zones in the 
+  [Compute Engine documentation](https://cloud.google.com/compute/docs/regions-zones/#available).
+  If you plan to use accelerators, make sure to pick a zone that supports the 
+  type you want.
+* `kfctl apply` attempts to fetch your email address from your 
```

Comment:
from gcloud; i.e User can run gcloud auth list to see what the default account is.

Reply:
Nice. Done.